app/models/form/question.rb

```diff
@@ -41,6 +53,16 @@ def enabled?(case_log)
     conditional_on.map { |condition| evaluate_condition(condition, case_log) }.all?
   end
 
+  def hidden_in_check_answers?
+    hidden_in_check_answers
+  end
```

Comment:
Could this just be attr accessor?

Reply:
I guess this way you get the `?` boolean return naming convention? Although I suppose current implementation doesn't _definitely_ return a boolean?